splines/gpuSplineUtils.cu

```diff
@@ -29,8 +29,8 @@
 #define __N_SPLINES__ NSPLINES_ND280
 // EM: for OA2024:
 #else
+#define __N_SPLINES__ 160
```

Comment:
There is obviously no way around this for now and only ND280 currently uses the gpuSplineUtils so this isn't a problem. But just a comment for myself to remember that we should think if there is some way around this?

Reply:
Easiest way is in T2K cmake set number of splines as argument you pass to cpm. This argument is being set in cmake and core will compile with some number.

This way each experiment will be able to have different number of splines but as long as number is known at compilation level we are fine.

Long term we might ocnsider to removing it completeyl but this might impact performance so will need some testing.